src/Schema.spec.js

```diff
@@ -500,12 +505,14 @@ describe('Schema', function () {
 
 
     schema.establishObservables(item);
-    expect(isObservable(item.brand)).toBe(true);
```

Comment:
how did those tests ever worked ?

Reply:
Because the content brand was a observable container. But brand as a property was not an observable. This is changed now.

Reply:
can we add a test that covers this to prevent this from happening again?

Reply:
the new test assertions cover that.

Reply:
ah i see